.github/workflows/beam_PreCommit_Website_Stage_GCS.yml

```diff
@@ -78,25 +78,8 @@ jobs:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           github_job: ${{ github.job }}
           github_current_run_id: ${{ github.run_id }}
-      - name: Find Current Pull Request
-        uses: actions/github-script@v6
-        id: findPr
-        with:
-          script: |
-            // Based on https://github.com/jwalton/gh-find-current-pr
-            const result = await github.rest.repos.listPullRequestsAssociatedWithCommit({
-                owner: context.repo.owner,
-                repo: context.repo.repo,
-                commit_sha: '${{ github.event.pull_request.head.sha || github.sha }}',
-            });
-            console.log(result);
-            const pr =
-                result.data.find((el) => {
-                    return context.payload.ref === `refs/heads/${el.head.ref}`;
-                }) || result.data[0];
-            const prNumber = (pr && pr.number) || context.event.issue.number;
-            console.log(prNumber);
-            core.setOutput('ghprbPullId', prNumber);
+      - name: Echo PR number
+        run: echo "ghprbPullId=${{ github.event.pull_request.number || github.event.issue.number }}" >> $GITHUB_ENV
```

Comment:
Oh yeah, I think this should just work; any idea on why we needed the action initially? I figured it was because the payload wasn't correct in some cases on PR update

Reply:
I added this action initially in order to get PR number on push trigger. However it wasn't needed, because when we have a PR, and we do push, pull_request_target will trigger anyway.